### DISCUSSION TITLE
tailwindcss 로 글자를 하얀색으로 바꾸고 싶은데 text-white가 안됩니다.

### DISCUSSION THREAD

**Discussion by Taewook1212** [FRONTENDSCHOOL8/popcorn-euid#38] [Q&A] [answered]
tailwindcss 로 글자를 하얀색으로 바꾸고 싶은데 text-white가 안됩니다.
class 의 순서에도 신경쓴것같은데 안되네요.
다른 방법이 있을지 궁금합니다

**Comment by asdfqaz74** [comment, marked as answer]
보통은 .text-white 가 맞는데
저희가 tailwind.config 에 text-background: '#ffffff' 라고 따로 정의해서

text-background 라고 하시면 됩니다!

**Reply by Taewook1212** [reply]
호옹 감사합니다! ㅎㅎ
